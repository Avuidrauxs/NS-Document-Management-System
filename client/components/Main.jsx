import React from 'react';
import { PropTypes } from 'prop-types';
import injectTapEventPlugin from 'react-tap-event-plugin';


injectTapEventPlugin();


// <Header> and <Footer>
const Main = props => (
  <div>
    { props.children }
  </div>
);

Main.propTypes = {
  children: PropTypes.array.isRequired
};

Main.defaultProps = {
  children: [],
};

export default Main;
