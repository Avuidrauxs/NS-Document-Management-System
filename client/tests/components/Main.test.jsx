import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import expects from 'expect';


import Main from '../../components/Main';

describe('Main', () => {
  it('should exist', () => {
    expects(Main).toExist();
  });
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Main />, div);
  });
  it('should render a snapshot', () => {
    const tree = renderer.create(<Main />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
