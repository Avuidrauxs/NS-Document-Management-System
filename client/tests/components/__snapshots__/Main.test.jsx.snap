// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should render a snapshot 1`] = `<div />`;
